### DISCUSSION TITLE
Shows only 8 items in dashboard creating with a zabbix plugin

### DISCUSSION THREAD

**Discussion by makiavellye32** [grafana/grafana#54617] [Dashboards, Panels & Visualizations]
Hi everyone.

I updated it from v8.2.3 to v9.1.2 and noticed that when creating a panel, the number of suggested parameters is limited to 8.
In the old version, the drop-down menu had a scrollbar, in the new version it doesn’t, so I can’t select anything other than the first 8 suggested options, and as a result, creating new panels is impossible.
Screenshot for example:

https://ibb.co/cvNnhGG

After that, I tried to put clean versions of grafani on Centos 7, 8. Ubuntu 22 and got the same effect there.
And then i connected my zabbix to old grafana and anytging was ok..
How to fix it? maybe I’m missing some modules in the OS? or is it a bug?

With respect. Ty for any advice

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
